doc/classes/Rect2i.xml

```diff
@@ -4,9 +4,10 @@
 		A 2D axis-aligned bounding box using integer coordinates.
 	</brief_description>
 	<description>
-		[Rect2i] consists of a position, a size, and several utility functions. It is typically used for fast overlap tests.
-		It uses integer coordinates. If you need floating-point coordinates, use [Rect2] instead.
-		Negative values for [member size] are not supported and will not work for most methods. Use [method abs] to get a Rect2i with a positive size.
+		The [Rect2i] built-in [Variant] type represents an axis-aligned rectangle in a 2D space, using integer coordinates. It is defined by its [member position] and [member size], which are [Vector2i]. Because it does not rotate, it is frequently used for fast overlap tests (see [method intersects]).
+		For floating-point coordinates, see [Rect2].
+		[b]Note:[/b] Negative values for [member size] are not supported. With negative size, most [Rect2i] methods do not work correctly. Use [method abs] to get an equivalent [Rect2i] with a non-negative size.
+		[b]Note:[/b] In a boolean context, a [Rect2] evaluates to [code]false[/code] if both [member position] and [member size] are zero (equal to [constant Vector2.ZERO]). Otherwise, it always evaluates to [code]true[/code].
```

Comment:
```suggestion
		[b]Note:[/b] In a boolean context, a [Rect2i] evaluates to [code]false[/code] if both [member position] and [member size] are zero (equal to [constant Vector2i.ZERO]). Otherwise, it always evaluates to [code]true[/code].
```